src/license_text_normalizer/main.py

```diff
@@ -24,6 +24,7 @@
     "*",
     "-",
     "@echo",
+    "=",
```

Comment:
Now that we're not relying on the delimiters for the new functionality, I think it would be better not to add this here. If at a later point we determine that `=` is a leading delimiter that should be stripped, then we should add it at that point.